www/wp-content/sunrise.php

```diff
@@ -50,10 +50,19 @@
 	die( 'The constant "COOKIE_DOMAIN" is defined (probably in wp-config.php). Please remove or comment out that define() line.' );
 }
 
-//Capture the domain and path from the current request
-$requested_domain    = $_SERVER['HTTP_HOST'];
+// Capture the domain and path from the current request
+$requested_domain = strtolower( $_SERVER['HTTP_HOST'] );
 $requested_uri       = trim( $_SERVER['REQUEST_URI'], '/' );
 
+// Strip any port numbers from the HTTP_HOST data.
+if ( substr( $requested_domain, -3 ) == ':80' ) {
```

Comment:
We should use Yoda conditions and strict comparisons here and on line 61. Otherwise, looks good!